voter-dapp/src/containers/VoteData.js

```diff
@@ -87,24 +87,29 @@ function useVoteData() {
           }
         }
 
+        // Data on unique users:
```

Comment:
The data fetched in this container doesn't actually change, I just refactor the setting of variables